python/cudf/dataframe/columnops.py

```diff
@@ -92,6 +92,17 @@ def fillna(self, fill_value):
         raise NotImplementedError
 
 
+def column_empty(data_size, dtype, masked):
```

Comment:
This is essentially the same function as `column_empty_like` but without the explicit tie to a given column. If the difference is just that we can make a column of arbitrary size, dtype, and yes/no mask, we should consider renaming this or refactor and rename `column_empty_like`.

`column_empty_like` doesn't actually need to be tied to a column, as it only uses the column explicitly to get a size value. Refactoring would require updating places it's used, though.

Reply:
renamed and unified `column_empty_like`.

Reply:
Now, the name of the function does not match its functionality. `column_empty_like` should probably not have the word `like` in it anymore. Or we can have another function called `column_empty_like` which just calls `column_empty`